exercises/protein-translation/example.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+module.exports = translate;
+
+function translate(rnaStrand) {
+  var proteins = [];
+
+  if (rnaStrand) {
+    for (var i = 0; i < rnaStrand.length; i+=3) {
+      var protein = getProtein(rnaStrand.substring(i, i + 3));
+
+      if (protein) {
+
+        if (protein === "STOP")
```

Comment:
We're using the AirBnb JS style guide on this track, so your `if` statements should have [braces please](https://github.com/airbnb/javascript#blocks--braces)!

Reply:
Fixed!

Reply:
Thanks @Smarticles101 !